apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -45,6 +45,7 @@ class SectionTable extends Component {
 
   render() {
     const { sectionIds } = this.props;
+    sectionIds.reverse();
```

Comment:
This is going to mutate props (sectionIds) which is a no-no in React land.  Best option is probably just to do `sectionIds.slice().reverse()` (makes a copy, and then reverses it).